sdk/aqueduct/utils.py

```diff
@@ -340,8 +344,26 @@ def _filter_out_blacklisted_requirements(packaged_requirements_path: str) -> Non
             f.write(line)
 
 
+def _add_cloudpickle_to_requirements(packaged_requirements_path: str) -> None:
+    """
+    Regardless of how we detect dependencies, we must include cloudpickle (with client's version
+    number) as a requirement because the server needs to install the same version of cloudpickle as
+    the client.
+
+    If the user-specified requirements file already contains a cloudpickle entry and the version
+    number matches, the installation process will still succeed. If there is a mismatch, the installation
+    will fail and the user should fix the version number to match the version installed on the client.
+    """
+    with open(packaged_requirements_path, "a") as f:
+        cloudpickle_requirement = (
+            "\ncloudpickle==%s" % pkg_resources.get_distribution("cloudpickle").version
+        )
+        f.write(cloudpickle_requirement)
+
+
 def _infer_requirements() -> List[str]:
-    """Obtains the list of pip requirements specifiers from the current python environment using `pip freeze`.
+    """
```

Comment:
I think it's time we settle on a comment style in python. The previous comment follows the Google style guide: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods, where the first comment is on the same line as `"""`. Can we keep this consistent? Either we keep it this way or change it all.